src/mangosd/WorldRunnable.cpp

```diff
@@ -118,7 +117,7 @@ void WorldRunnable::operator()()
     sBattleGroundMgr.DeleteAllBattleGrounds();
 
     sLog.Out(LOG_BASIC, LOG_LVL_MINIMAL, "Stopping network threads...");
-    sWorldSocketMgr->StopNetwork();
+    //sWorldSocketMgr->StopNetwork();
```

Comment:
is there no need to call stop network anymore?

Reply:
Will add `cancellationToken` in my next commit.

Currently its not possible to shutdown the network threads without stopping the whole `IoCtx`.
Each thread will get a `cancellationToken` which is just a volatile boolean, so they can be stopped without stopping the whole io context.